notebooks/04drop.py

```diff
@@ -0,0 +1,35 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:light
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.1
+#   kernelspec:
+#     display_name: 'Python 3.7.10 64-bit (''workflow-miniscope'': conda)'
+#     metadata:
+#       interpreter:
+#         hash: fb69438121b2de1073146525f02c3c959a56b9d2c1d6b0a62c34f01201323d1c
+#     name: python3
+# ---
+
+# # Drop schemas
+#
+# + This notebook is NOT required.
```

Comment:
This is a little confusing. We could say something like "Schemas are not typically get dropped in a production pipeline with real data in it. At the developmental phase, it might be required for the table redesign. When dropping all schemas is needed, the following is the dependency order."

Reply:
Great explanation.  Updated this notebook.